docs/whatsnew/0.2.0.rst

```diff
@@ -15,6 +15,9 @@ Breaking Changes
 
 Enhancements
 ~~~~~~~~~~~~
+* Updated private function :py:func:`~pvanalytics.features.daytime._correct_edge_of_day_errors`
+  to be more performant by vectorization; the original logic was using a lambda call that was
+  slowing the function speed down considerably. (:pull:`186`)
```

Comment:
Might be good to mention the relevant public function, since that's what most readers would care about. Also might be interesting to include a rough speed-up factor?

Reply:
Updated in https://github.com/pvlib/pvanalytics/pull/186/commits/d71fc15c3804b1129ca7ea1faf59607a65a935de